great_expectations/core/usage_statistics/anonymizers/types/base.py

```diff
@@ -50,3 +51,82 @@
     "runtime_configuration",
 }
+
+
+class CLISuiteInteractiveFlagCombinations(Enum):
+    UNPROMPTED_INTERACTIVE_FALSE_MANUAL_FALSE = {
+        "interactive_flag": None,
+        "interactive_attribution": "unprompted_interactive_absent_manual_absent",
+    }
+    UNPROMPTED_INTERACTIVE_TRUE_MANUAL_FALSE = {
+        "interactive_flag": True,
+        "interactive_attribution": "unprompted_interactive_present_manual_absent",
+    }
+    UNPROMPTED_INTERACTIVE_FALSE_MANUAL_TRUE = {
+        "interactive_flag": False,
+        "interactive_attribution": "unprompted_interactive_absent_manual_present",
+    }
+
+    UNPROMPTED_OVERRIDE_INTERACTIVE_FALSE_MANUAL_FALSE_PROFILE_TRUE = {
+        "interactive_flag": True,
+        "interactive_attribution": "unprompted_override_interactive_present_manual_absent_profile_present",
+    }
+    UNPROMPTED_OVERRIDE_INTERACTIVE_FALSE_MANUAL_TRUE_PROFILE_TRUE = {
+        "interactive_flag": True,
+        "interactive_attribution": "unprompted_override_interactive_absent_manual_present_profile_present",
+    }
+    UNPROMPTED_OVERRIDE_INTERACTIVE_FALSE_MANUAL_FALSE_BATCH_REQUEST_SPECIFIED = {
+        "interactive_flag": True,
+        "interactive_attribution": "unprompted_override_interactive_absent_manual_absent_batch_request_specified",
+    }
+    UNPROMPTED_OVERRIDE_INTERACTIVE_FALSE_MANUAL_TRUE_BATCH_REQUEST_SPECIFIED = {
+        "interactive_flag": True,
+        "interactive_attribution": "unprompted_override_interactive_absent_manual_present_batch_request_specified",
+    }
+    UNPROMPTED_OVERRIDE_INTERACTIVE_FALSE_MANUAL_FALSE_DATASOURCE_SPECIFIED = {
+        "interactive_flag": True,
+        "interactive_attribution": "unprompted_override_interactive_absent_manual_absent_datasource_specified",
+    }
+    UNPROMPTED_OVERRIDE_INTERACTIVE_TRUE_MANUAL_FALSE_DATASOURCE_SPECIFIED = {
+        "interactive_flag": True,
+        "interactive_attribution": "unprompted_override_interactive_present_manual_absent_datasource_specified",
+    }
+    UNPROMPTED_OVERRIDE_INTERACTIVE_FALSE_MANUAL_TRUE_DATASOURCE_SPECIFIED = {
+        "interactive_flag": True,
+        "interactive_attribution": "unprompted_override_interactive_absent_manual_present_datasource_specified",
+    }
+
+    PROMPTED_CHOICE_DEFAULT = {
+        "interactive_flag": False,
+        "interactive_attribution": "propmpted_choice_false_default",
+    }
+    PROMPTED_CHOICE_FALSE = {
+        "interactive_flag": False,
+        "interactive_attribution": "prompted_choice_false",
+    }
+    PROMPTED_CHOICE_TRUE_PROFILE_FALSE = {
+        "interactive_flag": True,
+        "interactive_attribution": "prompted_choice_true_profile_false",
+    }
+    PROMPTED_CHOICE_TRUE_PROFILE_TRUE = {
+        "interactive_flag": True,
+        "interactive_attribution": "prompted_choice_true_profile_true",
+    }
+    PROMPTED_CHOICE_TRUE = {
+        "interactive_flag": True,
+        "interactive_attribution": "prompted_choice_true",
+    }
```

Comment:
I think it would help to add `interactive_` before the boolean in the attribution to make it a bit more clear:
```suggestion
    PROMPTED_CHOICE_DEFAULT = {
        "interactive_flag": False,
        "interactive_attribution": "propmpted_choice_interactive_false_default",
    }
    PROMPTED_CHOICE_FALSE = {
        "interactive_flag": False,
        "interactive_attribution": "prompted_choice_interactive_false",
    }
    PROMPTED_CHOICE_TRUE_PROFILE_FALSE = {
        "interactive_flag": True,
        "interactive_attribution": "prompted_choice_interactive_true_profile_false",
    }
    PROMPTED_CHOICE_TRUE_PROFILE_TRUE = {
        "interactive_flag": True,
        "interactive_attribution": "prompted_choice_interactive_true_profile_true",
    }
    PROMPTED_CHOICE_TRUE = {
        "interactive_flag": True,
        "interactive_attribution": "prompted_choice_interactive_true",
    }
```